errorHandling/error.js

```diff
@@ -0,0 +1,15 @@
+exports.errorHandlerOne = (err, res) => {
+    console.log('Error Handler 1')
+    
+    res.status(404).send({msg: 'Not Found' });
+
+    
+  }
+
+
+
+
+exports.errorHandlerTwo = (err, req, res, next) => {
```

Comment:
We should probably give these functions descriptive names to make it easier to follow the flow of your code.